source/production/F0.Minesweeper.Components/Cell.razor.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Threading.Tasks;
+using F0.Minesweeper.Logic.Abstractions;
+using Microsoft.AspNetCore.Components;
+
+namespace F0.Minesweeper.Components
+{
+	public partial class Cell
+	{
+		[Parameter]
+		public Location Location { get; set; }
+		private string StatusText
+		{
+			get => statusText;
+			set
+			{
+				if (statusText == value)
+				{
+					return;
+				}
+
+				statusText = value;
+			}
+		}
+
+		private string statusText;
+
+		public Cell()
+		{
+			StatusText = "Covered";
+		}
+
+		protected override void OnParametersSet()
+		{
+			if (Location == null)
```

Comment:
Since `Location` is a _record type_, for which the compiler synthesizes a `operator ==`, we should use _pattern matching_ rather than the _equality operator_ to check for _null_. That way, we directly check for _null_, rather than detouring via `==` and `Equals`.

```suggestion
			if (Location is null)
```